core/app/assets/js/ui/atoms/text.js

```diff
@@ -3,19 +3,28 @@ export default function Text( props ) {
 		classes = [
 			props.className,
 		],
-		tagName = props.tag,
 		variant = props.variant && 'md' !== props.variant ? '-' + props.variant : '';
 
 	classes.push( baseClassName + '-text' + variant );
 
-	return React.createElement( tagName, {
-		className: classes.filter( ( classItem ) => classItem ).join( ' ' ),
-	}, props.children );
+	const classesString = Utils.arrayToClassName( classes );
+
+	if ( props.tag ) {
+		return React.createElement( props.tag, {
+			className: classesString,
+		}, props.children );
+	}
+
+	return (
+		<p className={ classesString }>
```

Comment:
`defaultProps.tag=p`

Reply:
The storybook doesn't know how to get the component props when no element is returned. I understand it's unusual, but I think we need to consider it for the benefit of getting the component info automatically.

Reply:
Fixed.